docroot/sites/its.uiowa.edu/modules/its_core/src/Form/ItsCoreSettingsForm.php

```diff
@@ -116,6 +115,15 @@ public function validateForm(array &$form, FormStateInterface $form_state) {
         }
       }
     }
+
+    // Validate 'single-alert-secondary' as a single email.
```

Comment:
```suggestion
    // Validate 'single-alert-secondary' as a single email as we want it to act like a BCC and not reveal additional recipients.
```

Just as time goes on, the BCC desire of this functionality might get lost in the commit history.